<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>68</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>419</width>
    <height>68</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>419</width>
    <height>68</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Focus</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(110, 110, 110, 255), stop:0.101942 rgba(91, 91, 91, 255), stop:0.820388 rgba(60, 60, 60, 255), stop:0.990291 rgba(37, 37, 37, 255));
	color: rgb(225, 225, 225);
}
QTextBrowser, QLineEdit {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(37, 37, 37, 147), stop:0.0873786 rgba(60, 60, 60, 150), stop:0.932039 rgba(91, 91, 91, 148), stop:1 rgba(110, 110, 110, 148));
	border: 1px  solid rgb(147, 45, 203);
	border-radius: 3px;
}

QLCDNumber, QSpinBox, QDateTimeEdit {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(37, 37, 37, 147), stop:0.0873786 rgba(60, 60, 60, 150), stop:0.932039 rgba(91, 91, 91, 148), stop:1 rgba(110, 110, 110, 148));
	border: 1px  solid rgb(44, 44, 44);
	border-radius: 3px;
}

QLabel {
	background-color:rgba(255, 255, 255, 0);
}

QPushButton {
	background-color:rgba(255, 255, 255, 0);
}

QProgressBar {
	background-color: rgb(44, 44, 44);
	border: 1px  solid rgb(44, 44, 44);
	margin: 8px, 8px;
	color:rgba(255, 255, 255, 0);
}
QProgressBar::chunk {
	background-color: rgb(147, 45, 203);
}</string>
  </property>
  <widget class="QProgressBar" name="Progress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="TimeDone">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="TimeLeft">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="DoneButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BreakButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Break</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="TagsArea">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
